### PR TITLE
Test error handling / notifications via CloudWatch alerts

### PR DESCRIPTION
**Pull request by caggle**

### CODE REVIEW

File: handler.py [hunk 1]
```diff
@@ -9,3 +9,6 @@ def run(event, context):
     current_time = datetime.datetime.now().time()
     name = context.function_name
     logger.info("Your cron function " + name + " ran at " + str(current_time))
+    
+    # trigger error here
+    i = 7 / 0
```
Comment (claudijd) [inline handler.py]: Do we want to leave this logic bug in place or remove it, assuming a natural error will be escalated to notifications?

File: serverless.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-service: aws-python-scheduled-cron
+service: scheduled-cron-error-handling
```
Comment (claudijd) [inline serverless.yml]: Maybe leave this unchanged until we pick a project name for the template?

Comment (tristanweir) [reply serverless.yml]: This was to avoid overwrites of other peoples deployment/testing

File: handler.py (hunk 1 above)
Comment (caggle) [reply handler.py]: Maybe leave it for now, until we find a way to make it actually work? Then we can update it with a subsequent commit/PR.

**Comment by pwnbus** [comment]
Closing since we implemented in https://github.com/claudijd/serverless-cron-template/pull/9

**Pull request closed without merging**